llvm/test/CodeGen/AIE/aie2/GlobalISel/inst-select-no-combine.mir

```diff
@@ -74,8 +74,8 @@ body:             |
     ; CHECK-NEXT: {{  $}}
     ; CHECK-NEXT: [[COPY:%[0-9]+]]:vec256 = COPY $wl0
     ; CHECK-NEXT: [[COPY1:%[0-9]+]]:ep = COPY $p0
-    ; CHECK-NEXT: [[PADD_imm_pseudo:%[0-9]+]]:ep = PADD_imm_pseudo [[COPY1]], 992
-    ; CHECK-NEXT: VST_dmw_sts_w_ag_idx_imm [[COPY]], [[PADD_imm_pseudo]], 0 :: (store (<8 x s32>))
```

Comment:
Would it reduce the diff noise if we kept PADD_imm_pseudo for the most versatile ABS case?